package.json

```diff
@@ -17,39 +17,47 @@
     "react",
     "react-native",
     "jsx",
-    "js"
+    "js",
+    "javascript",
+    "ts",
+    "tsx",
+    "typescript"
   ],
   "author": "typedigital",
   "license": "Unlicense",
   "scripts": {
-    "lint": "npx eslint . --config=base.js",
-    "lint-fix": "npx eslint . --config=base.js --fix",
-    "test": "eslint ./tests/*.js --config=base.js",
+    "lint": "eslint ./*.js --config=base.js",
+    "lint-fix": "eslint ./*.js --config=base.js --fix",
+    "test": "npm-run-all test:*",
     "test:base": "eslint ./tests/base/*.js --max-warnings 0 --config=base.js",
     "test:typescript": "eslint tests/typescript/*.ts --max-warnings 0 --config=tests/typescript/.eslintrc",
-    "release": "standard-version"
+    "release": "standard-version",
+    "find-unused": "eslint-find-rules -u -n -d ./base.js",
+    "find-unused:ts": "eslint-find-rules -u -n -d ./tests/typescript/.eslintrc"
```

Comment:
same here.
What are the criteria for a rule being unused? I'm asking cause e.g. "no-precision-loss" is set to "warn" but it's being labeled "unused".

Reply:
We should investigate that would you open an investigation ticket? @JoeTypeDigital